contracts/Comptroller/Diamond/facets/PolicyFacet.sol

```diff
@@ -499,4 +533,36 @@ contract PolicyFacet is IPolicyFacet, XVSRewardsHelper {
             emit VenusBorrowSpeedUpdated(vToken, borrowSpeed);
         }
     }
+    /// @notice Sets the keyring configuration.
+    /// @param _keyringChecker The address of the keyring checker.
+    /// @param _keyringPolicyId The policy id of the keyring.
+    function setKeyringConfiguration(address _keyringChecker, uint32 _keyringPolicyId) public {
+        ensureAdmin(msg.sender);
+        if (address(_keyringChecker) == address(0) || _keyringPolicyId == 0) {
+            revert InvalidKeyringConfiguration(address(_keyringChecker), _keyringPolicyId);
+        }
+        keyringChecker = IKeyringChecker(_keyringChecker);
+        keyringPolicyId = _keyringPolicyId;
+        emit KeyringConfigurationUpdated(address(_keyringChecker), _keyringPolicyId);
```

Comment:
I think we don't need the cast to `address`, because `_keyringChecker` is already an address